api/bazel/external_proto_deps.bzl

```diff
@@ -17,20 +18,23 @@ EXTERNAL_PROTO_IMPORT_BAZEL_DEP_MAP = {
 
 # This maps from the Bazel proto_library target to the Go language binding target for external dependencies.
 EXTERNAL_PROTO_GO_BAZEL_DEP_MAP = {
+    "@com_google_googleapis//google/api/expr/v1alpha1:checked_proto": "@com_google_googleapis//google/api/expr/v1alpha1:expr_go_proto",
```

Comment:
Ditto, why are we seeing reformats here?

Reply:
As explained above, proto has two libraries, but only has one go_proto.